lib/elm-files.js

```diff
@@ -52,6 +53,9 @@ async function getProjectFiles(options, elmSyntaxVersion) {
   const emptyDirectories = filesInDirectories.filter(
     ({files}) => files.length === 0
   );
+  const elmFilesToRead = [
+    ...new Set(filesInDirectories.flatMap((directory) => directory.files))
+  ];
```

Comment:
Just noticed that is just unique() inlined. Maybe I'll de-inline it someday. Probably not. :)

Reply:
Ah right. Yeah I reached out for `unique` after writing this code. I'll make the change.

Reply:
I was rebasing and this made a conflict, so I fixed it in #344.

Reply:
Just saw your message. I fixed it in #354, it's on master now :sweat_smile:

Reply:
Already re-(re-?)rebased